alpa/collective/mesh_run_nccl_collective.py

```diff
@@ -0,0 +1,390 @@
+"""Implementation of devive_mesh's send/recv/allgather/broadcast."""
+
+from typing import Sequence
+import logging
+
+from jax import device_put
+from jax._src.lib import xla_extension as xe
+
+import jax.numpy as jnp
+import numpy as np
+import cupy
+
+import alpa.collective as col
+from alpa.collective.collective_group import nccl_util
+from alpa.util import (jax_tensor_to_cupy, cupy_to_jax_tensor, jax_tensor_set,
+                       xla_buffer_to_jax_tensor, jax_tensor_to_xla_buffer,
+                       xla_buffer_to_cupy, cupy_to_xla_buffer,
+                       is_continuous_subset, infer_offset_and_n_elements,
+                       jax_tensor_index, infer_start_pos_and_n_elements)
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def xla_nccl_send_tile(mesh, uuid: int, offset: Sequence[slice], dst_rank: int,
```

Comment:
Rename all `mesh` to `worker` in this file. It is a single worker not a mesh of workers.

Reply:
OK